File: 20181027examples/ltvtvpmcprinter/src/main/java/lt/vtvpmc/printer/shapes/Circle.java
package lt.vtvpmc.printer.shapes;

import lt.vtvpmc.printer.Shape;

public class Circle implements Shape {
    
    private double r;
    
    public Circle(double r) {
        this.r = r;
    }
    
    @Override
    public double getArea() {
        return Math.PI * this.r * this.r;
    }
    
}

File: VCS/PirmaPamoka/SestaPamoka/Circle.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SestaPamoka
{
    class Circle
    {
        private double _radius = 1;     // savybe (su reiksme kuri bus naudojamas jei nebus ivesta kitokiu ar jokiu)
        private string color = "red";   

        public Circle()                 // konstruktorius (neuzduodantis reiksmiu ivedimo)
        {

        }

        public Circle (double radius)   // konstruktorius (uzduodantis reiksmes ivedima)
        {
            _radius = radius;
        }

        public double GetArea()         // metodas
        {
            return Math.PI * Math.Pow(_radius, 2);
        }

        public double GetLength()       // metodas
        {
            return 2 * Math.PI * _radius;
        }
    }
}
